audmath/core/api.py

```diff
@@ -78,6 +87,152 @@ def db(
     return x
 
 
+def window(
+    samples: int,
+    shape: str = 'tukey',
+    half: str = None,
+    offset: typing.Union[int, float] = 0,
```

Comment:
Since we have an `offset` would it also make sense to have an option to set the peak?

Reply:
Mh, good question.
I think I would rather remove the `offset` argument and leaf the scaling to the user as it is trivial anyway, e.g.
```
win * (1. - offset) + offset
```

Reply:
Agree, let's keep it simple.

Reply:
I removed the `offset` argument.